libbeat/outputs/kafka/kafka_integration_test.go

```diff
@@ -23,7 +24,198 @@ const (
 	kafkaDefaultPort = "9092"
 )
 
-var testOptions = outputs.Options{}
+func TestKafkaPublish(t *testing.T) {
+	single := modetest.SingleEvent
+	// multi := modetest.MultiEvent
```

Comment:
they can be removed as not being used. events of type []modetest.EventInfo hold a number of events + setting to bulk or not bulk publish those events. It's used to drive sending events through output plugin.

In test randMulti is used.